tweepy/auth.py

```diff
@@ -117,8 +124,8 @@ class OAuthHandler(OAuth1UserHandler):
         Use :class:`OAuth1UserHandler` instead.
     """
 
-    def __init__(self, consumer_key, consumer_secret, access_token=None,
```

Comment:
This is probably not that important, but maybe we could use a better a way of deprecating old name classes here. See [this comment](https://stackoverflow.com/questions/9008444/how-to-warn-about-class-name-deprecation#comment80382529_26130976) on why it could be problematic, and a few options given in the SO thread